.eslintrc

```diff
@@ -23,9 +23,6 @@
         "no-dupe-keys": 2,
         "no-dupe-args": 2,
         "no-else-return": 2,
-        "no-empty": 2,
-        "no-empty-character-class": 2,
-        "no-empty-label": 2,
```

Comment:
According to the changelog this rule has been replaced by `no-labels`  with {"allowLoop": true, "allowSwitch": true} option.